Content.Shared/Emag/Components/EmagComponent.cs

```diff
@@ -17,4 +18,18 @@ public sealed partial class EmagComponent : Component
     [DataField("emagImmuneTag", customTypeSerializer: typeof(PrototypeIdSerializer<TagPrototype>)), ViewVariables(VVAccess.ReadWrite)]
     [AutoNetworkedField]
     public string EmagImmuneTag = "EmagImmune";
+
+    // DeltaV - Add a whitelist/blacklist to the Emag
+    /// <summary>
+    /// Whitelist that entities must be on to work.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+    public EntityWhitelist? Whitelist = null;
+
+    /// <summary>
+    /// Blacklist that entities must be off to work.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+    public EntityWhitelist? Blacklist = null;
```

Comment:
dont need ViewVariables, AutoNetworkedField or = null:
- datafield is automatically rw
- nothing ever changes them so they never get dirtied so they never get sent
- null is the default